docs/source/examples/checks/performance/simple_model_comparison.ipynb

```diff
@@ -5,118 +5,87 @@
    "id": "840ea662",
```

Comment:
<p>Title:</p><p>What is the purpose of the simple model check?</p><p> </p><p>and add another sentence about the "motivation" to compare to a baseline.... (basically - your two first sentences are great, would just elabaorate them a bit more... make them ~3)</p><p> </p><p>The second paragraph - its not very clear what is the default behavior, and what/how can the user choose...</p><p>For example after reading I don't understand if what the check does is "runs the three options", then chooses the highest scoring ones, or alternatively it "looks at my data, label and model type and heuristically chooses which model to train"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>